test/peer.cpp

```diff
@@ -62,7 +62,7 @@ BOOST_AUTO_TEST_CASE(save_nodes)
 	std::list<Host*> hosts;
 	for (auto i:{0,1,2,3,4,5})
 	{
-		Host* h = new Host("Test", NetworkPreferences(30300 + i, "127.0.0.1", false, true));
+		Host* h = new Host("Test", NetworkPreferences("127.0.0.1", 30300 + i, false));
```

Comment:
probably not for this PR, but trying to evict all bool params these days. make an `enum class` for this bool param at some point.